modules/standard/IO.chpl

```diff
@@ -3043,6 +3043,389 @@ record DefaultDeserializer {
   }
 }
 
+/*
+  A basic binary Serializer to be used with ``fileWriter``.
```

Comment:
I might remove the word "basic" here. It seems to have a slightly negative connotation.

Possibly replace with "direct"?

Reply:
How about "unstructured"?

Reply:
That sounds good 👍